build.gradle

```diff
@@ -18,6 +18,17 @@ allprojects {
 }
 // Define versions in a single place
 ext {
+    // Sdk and tools
+    minSdkVersion = 19
+    targetSdkVersion = 28
+    compileSdkVersion = 28
+
+    // App dependencies
+    supportLibraryVersion = '28.0.0'
```

Comment:
29.0.0?

Reply:
the latest supportLibraryVersion available is 28.0.0